### DISCUSSION TITLE
Script can't seem to find an extremely popular album

### DISCUSSION THREAD

**Discussion by Kamryx** [RandomNinjaAtk/docker-lidarr-extended#140] [Q&A] [answered]
So, everything seems to work pretty well for me, except the script can't seem to find Take Care by Drake, which is ironically one of the most popular albums of all time, let alone from his discography. 

Do I have something misconfigured? I can't figure out why it would struggle with this when it's managed to grab thousands of way more obscure albums. 

Also, a little add on question; is the script still searching for all of the other missing music in my Lidarr in this log I provided, or at all? I deleted and re-added this album to try and grab it again, and I assume the script recognizes that it hasn't been searched for, hence why it's spelled out so notably, but the other thousands of albums aren't. I again assume it knows it's searched previously for them and therefore doesn't spell them all out in a verbose manner, but maybe runs a background search or something, just in case they've popped up. 

[log.txt](https://github.com/RandomNinjaAtk/docker-lidarr-extended/files/10095313/log.txt)

**Comment by RandomNinjaAtk** [comment, marked as answer]
It doesn't search again for albums already previously searched for. If you read the log, you can see where it says it's filtering the results and what log files to delete if you want to go search them again.

For info on search/rejections: https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Audio-Script-Information#q-how-are-albums-rejected

**Reply by Kamryx** [reply]
Very late reply, but would it possible to add a setting that gives a threshold to actually re-search for a missing album? Say, if album was released within the last (x) days, search again. 

In my setup, it's basically not grabbing any new albums now, I assume because it's searching right when an album supposedly comes out, Deezer doesn't actually have it yet (I've seen releases be days late), therefore it thinks it's unfindable and gets thrown in the bin. Having to manually dig into my files and delete logs every time it happens is incredibly inconvenient and often impossible when I'm away from home. Seems like a super useful and easy option to resolve it.

**Reply by RandomNinjaAtk** [reply]
The script does retry notfound items on a time interval (days), and you can adjust it, see the parameters: https://github.com/RandomNinjaAtk/docker-lidarr-extended#parameters

Hint `retryNotFound`

**Reply by RandomNinjaAtk** [reply]
@Kamology This change: https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/8e70a8173245a65337ad1f9b9d59f181d22b2f12

Should do what your asking and resolve it....

Keep in mind, it hasn't had much testing, so may need a bug fix or adjustment...

**Reply by Kamryx** [reply]
Ay nice thank you! Yeah I was gonna say the retryNotFound is nice and useful but it'd probably still be worth having both for the somewhat distinct use cases. Should improve reliability a lot, I'll wait for some new releases and report back if I need to.
